meta-flex-support/recipes-core/images/development-image.bb

```diff
@@ -5,11 +5,9 @@
 SUMMARY = "A development/debugging image that fully supports the target \
 device hardware."
 
-IMAGE_FEATURES = "${IMAGE_FEATURES_DEVELOPMENT} ${EXTRA_IMAGE_FEATURES}"
-IMAGE_FEATURES_DEVELOPMENT ?= "debug-tweaks"
-IMAGE_FEATURES_DEVELOPMENT:append:feature-tracing = " tools-profile"
-IMAGE_FEATURES_DISABLED_DEVELOPMENT ?= ""
-IMAGE_FEATURES:remove = "${IMAGE_FEATURES_DISABLED_DEVELOPMENT}"
+IMAGE_FEATURES = "${EXTRA_IMAGE_FEATURES}"
```

Comment:
We don't have to do this manually, it's taken care of in the core layer. Just populate IMAGE_FEATURES here with a weak assignment so user can override those if needed.

Reply:
@Noor-Ahsan let's merge this PR and I'll fix this in #187